utils/spacewalk-common-channels.ini

```diff
@@ -3020,3 +3020,165 @@ gpgkey_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni
 gpgkey_id = %(_uyuni_gpgkey_id)s
 gpgkey_fingerprint = %(_uyuni_gpgkey_fingerprint)s
 repo_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Master:/EL8-Uyuni-Client-Tools/EL_8/
+
+[raspbian-9-pool-armhf-uyuni]
```

Comment:
Debian 9 is only on LTS and goes 100% EoL next year (https://wiki.debian.org/DebianReleases). Do we really want to add it?

Raspbian is now renamed: https://www.raspberrypi.org/software/ Shouldn't we use the new name?

Also remember we need to add client tools repos as well (can come later)

Reply:
Raspberry Pi OS is for x86 only, it's not the OS for Raspberry Pi itself

Reply:
About the version I don't really know if we need to add version 9, but there is no much difference in case of Salt Bundle, and venv-salt-minion was build for version 9 already.

Reply:
> Raspberry Pi OS is for x86 only, it's not the OS for Raspberry Pi itself

I think you are mixing the image installer with the OS?

Raspberry Pi OS is indeed the former Raspbian:


> Your Raspberry Pi needs an operating system to work. This is it. Raspberry Pi OS (previously called Raspbian) is our official supported operating system.
> https://www.raspberrypi.org/software/operating-systems/#raspberry-pi-os-32-bit

Reply:
> About the version I don't really know if we need to add version 9, but there is no much difference in case of Salt Bundle, and venv-salt-minion was build for version 9 already.

Ok, let's add it, and we'll see.

Reply:
Oh, sorry. I see now `Raspberry Pi OS`. Some time ago I remember this name was used for this only https://www.raspberrypi.org/software/raspberry-pi-desktop/
I just missed the plans to change the name for the distro. Not sure which one is better to use in such case. Probably it's better to use Raspbian as it was originally named this way.

Reply:
I am not sure. Following that rule could mean people is still calling us `SuSE` :-)

I'd suggest we use the official naming at the code and the doc, even if day-to-day we still call it "rasbpian".

Reply:
Hint: the salt grains shows "Raspbian"
https://github.com/saltstack/salt/blob/master/salt/grains/core.py#L1555

Reply:
Raspbian is everywhere in the distro, os-release also shows Raspbian. I think it's better to add these channels as `raspbian-...` as most of the users know it with this name and the distro is also self identifying as Raspbian